src/help/zaphelp/contents/ui/dialogs/options/api.html

```diff
@@ -25,11 +30,28 @@
 The API key is used to prevent malicious sites from accessing the ZAP API.<br/>
 It is strongly recommended that you set a key unless you are using ZAP in a completely isolated environment.<br/>
 
+<H3>IP addresses permitted to use the API</H3>
+
+By default only the machine ZAP is running on is able to access the ZAP API.<br/>
+You can allow other machines access to the API by adding suitable regex patterns.<br/>
+You should only add IP addresses that you trust.
+
 <H3>Disable the API Key</H3>
 
 Selecting this option disables the API key.<br/>
 This is not recommended unless you are using ZAP in a completely isolated environment, as it allows malicious sites to access the ZAP API.
 
+<H3>Do not require an API key for safe operations</H3>
+
+If enabled then the API key is not required for Views or Other operations that are considered 'safe', in other words operations that do not make any changes to ZAP. 
+Such operations do however give access to ZAP data such as alert, messages and file system paths.  
```

Comment:
I support the serial (or oxford) comma, I'd suggest: `data such as alert, messages, and file system paths.`

"messages and file system paths" aren't a single entity, they're distinct.